fake_filesystem_test.py

```diff
@@ -3250,5 +3250,65 @@ def testExistsWithMixedSeparators(self):
     self.assertTrue(self.filesystem.Exists('?foo?bar?xyzzy?plugh'))
 
 
+class DiskSpaceTest(TestCase):
+  def setUp(self):
+    self.filesystem = fake_filesystem.FakeFilesystem(path_separator='/', total_size=1024)
+
+  def testFileSystemSizeAfterFileCreation(self):
```

Comment:
File contents are commonly strings.

Please add a test trying the ASCII string 'xyzzy', I suppose that this should consume:

> 5 characters + null terminator = 6 bytes

Then add a test trying the Unicode string '複雑' which means "complicated" in Japanese and Chinese.  I would think this would consume:

> 3 bytes for 複 + 3 bytes for 雑 + null terminator = 7 bytes